addons/planner/static/src/js/planner.js

```diff
@@ -192,13 +196,14 @@ var PlannerDialog = Widget.extend({
         this.$(".oe_planner li a[href=#"+page_id+"]").parent().addClass('active');
         this.$(".oe_planner div[id^='planner_page']").removeClass('show');
         this.$(".oe_planner div[id="+page_id+"]").addClass('show');
+        this.$(".oe_planner .pages").scrollTop("0");
         this.planner.data['last_open_page'] = page_id;
         session.set_cookie(this.cookie_name, page_id, 8*60*60); // create cookie for 8h
     },
     // planner data functions
     _get_values: function(page_id){
         // if no page_id, take the complete planner
-        var base_elem = page_id ? this.$el : this.$(".oe_planner div[id='planner_page"+page_id+"']");
+        var base_elem = page_id ? this.$(".oe_planner div[id="+page_id+"]") : this.$(".oe_planner div[id^='planner_page']");
```

Comment:
Oay, so the native behavior don't change very mush : without a 'page_id', we return all values of every pages of the planner ?
If so, I'm okay with that ! ;)